code/modules/projectiles/projectile/bullets.dm

```diff
@@ -198,8 +198,9 @@
 
 /obj/item/projectile/bullet/meteorshot/on_hit(var/atom/target, var/blocked = 0)
 	..()
-	if(istype(target, /atom/movable))
-		var/atom/movable/M = target
+	var/atom/movable/M = target
+	if(M.anchored)return
```

Comment:
Is there any particular reason to make this a one-liner? It makes the code really awkward to read.